hypha/static_src/src/sass/apply/base/_typography.scss

```diff
@@ -45,32 +42,39 @@ h6,
 h1,
 %h1,
 .alpha {
-    @include responsive-font-sizes(40px, map-get($font-sizes, alpha));
-    font-weight: $weight--black;
+    font-size: 2.25rem;
+    line-height: 2.5rem;
+    font-weight: 400;
```

Comment:
Why not use the variables for weight here as well?

Reply:
I was thinking to move and refractor these in a more meaningful and thoughtful way in another PR. Let me know your thoughts.